docs/user-interface/graphics/blendmodes.md

```diff
@@ -96,22 +96,22 @@ The `PlusLighter` blend mode sums the source and destination. Then, for values a
 
 The separable blend modes alter the individual red, green, and blue color components of a graphical object.
 
-The following table shows the separable blend modes, with brief explanations of what they do. In the table, *Dc* and *Sc* refer to the destination and source colors, and the second column shows the source color that produces no change:
-
-| Blend mode | No change | Operation |
-| -- | -- | -- |
-| `Normal` |  | No blending, source selected |
-| `Multiply` | `White` | Darkens by multiplying Sc-Dc |
-| `Screen` | `Black` | Complements the product of complements: Sc + Dc - Sc.Dc |
-| `Overlay` | `Gray` | Inverse of `HardLight` |
-| `Darken` | `White` | Minimum of colors: min(Sc, Dc) |
-| `Lighten` | `Black` | Maximum of colors: max(Sc, Dc) |
-| `ColorDodge` | `Black` | Brightens the destination based on the source |
-| `ColorBurn` | `White` | Darkens the destination based on the source |
-| `SoftLight` | `Gray` | Similar to the effect of a soft spotlight |
-| `HardLight` | `Gray` | Similar to the effect of a harsh spotlight |
-| `Difference` | `Black` | Subtracts the darker color from the lighter color: Abs(Dc - Sc) |
-| `Exclusion` | `Black` | Similar to `Difference` but lower contrast |
+The following table shows the separable blend modes, with brief explanations of what they do. In the table, `Dc` and `Sc` refer to the destination and source colors, and the second column shows the source color that produces no change:
+
+| Blend mode   | No change | Operation                                                         |
+|--------------|-----------|-------------------------------------------------------------------|
+| `Normal`     |           | No blending, source selected                                      |
+| `Multiply`   | `White`   | Darkens by multiplying `Sc-Dc`                                      |
```

Comment:
Can you check the "operation" column for line 104? It didn't make sense to me.

Reply:
Yep, it's fine. Graphics folks should understand it.